deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml

```diff
@@ -0,0 +1,194 @@
+{{ if not .Values.config.configmap.name }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-config
+  labels:
+    {{- include "venafi-kubernetes-agent.labels" . | nindent 4 }}
+data:
+  config.yaml: |-
+    server: {{ .Values.config.server | quote }}
+    venafi-cloud:
+      uploader_id: "no"
```

Comment:
Passing by: why is this set to `no`? Is that mandatory to set it to `no`, or is it a dummy value?

Reply:
Looking at the backend's codebase, in [upload.go](https://gitlab.com/venafi/vaas/applications/tls-protect-for-k8s/cloud-services/-/blob/f2a11dccf9ff8ac8c8af983e3947595043966c79/backend/internal/tlspkmodule/uploadserver/upload.go#L64):

```go
upload.POST("/upload/clusterdata/:uploaderID", s.StreamUploadHandler)
```

But looking down, this path segment isn't referenced anywhere... instead, `uploaderID` is set using the JWT's `sub` field:

```go
uploaderID, ok = ctx.Value(authm.SubjectKey).(string)
```

Is this path segment useless, and is this why you set `uploader_id` to `no`? I am utterly confused